backend/src/main/java/com/tourplanner/backend/controller/impl/TourController.java

```diff
@@ -0,0 +1,51 @@
+package com.tourplanner.backend.controller.impl;
+
+import com.tourplanner.backend.controller.IGenericController;
+import com.tourplanner.backend.service.dto.TourDTO;
+import com.tourplanner.backend.service.IGenericService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/tour")
+public class TourController implements IGenericController<TourDTO, Long> {
+
+
+    private final IGenericService<TourDTO, Long> tourService;
+
+    @Override
+    @PostMapping
+    public ResponseEntity<Void> create(@RequestBody TourDTO tourDTO) {
+        tourService.create(tourDTO);
+        return ResponseEntity.ok().build();
+    }
+    @Override
+    @GetMapping
+    public ResponseEntity<List<TourDTO>> findAll() {
+        return ResponseEntity.ok(tourService.findAll());
+    }
+
+    @Override
+    @GetMapping("/find{id}")
```

Comment:
Here it should be just "/{id}", so that the entire URI is like: GET /api/v1/tours/1 for example, which aligns with REST best practices.